src/ClientProtos/capture_data.proto

```diff
@@ -142,6 +142,7 @@ message TimerInfo {
   uint64 api_async_scope_id = 15;
   uint64 address_in_function = 16;
   string api_scope_name = 17;
+  uint64 api_scope_group_id = 18;
```

Comment:
Ouh. We already have `group_id`, which is a member of ApiScope's. Not 100% sure about a good name though.

So we are thinking of something that complements `function_id`, for manual instrumentation scopes. It can't be `...scope_id`, as we use scope for a single timer as well as for the scope macro. Maybe naively `manual_instrumentation_id`? Not sure though....

Reply:
I like `manual_instrumentation_id. @dpallotti WDYT?

Reply:
This is still open, right?